test/DeploymentPipelines.test.ts

```diff
@@ -44,18 +44,21 @@ describe('DeploymentPipelines', () => {
               config: {
                 activeTracing: Tracing.ACTIVE,
               },
+              stageType: TestStage,
             },
           ],
         },
       ],
+      pipelineConfig: {},
+      repository: {
+        source: {
+          codeCommitArn: 'arn:codecommit:us-east-1:us-east-1:123456789012:yes',
```

Comment:
This ARN, albeit just a test/dummy value seems to be incorrectly formatted. If this test is in fact passing, we should make sure this is being validated somewhere/somehow